src/main/java/uk/ac/ebi/ena/readtools/loader/fastq/DataSpot.java

```diff
@@ -22,9 +22,9 @@
 import uk.ac.ebi.ena.readtools.loader.common.InvalidBaseCharacterException;
 import uk.ac.ebi.ena.readtools.loader.common.QualityNormalizer;
 import uk.ac.ebi.ena.readtools.loader.common.QualityNormalizer.QualityNormaizationException;
-import uk.ac.ebi.ena.readtools.loader.common.feeder.DataFeederException;
-import uk.ac.ebi.ena.readtools.loader.common.feeder.FeedableData;
-import uk.ac.ebi.ena.readtools.loader.common.feeder.FeedableDataChecker;
+import uk.ac.ebi.ena.readtools.loader.common.feeder.DataProducerException;
+import uk.ac.ebi.ena.readtools.loader.common.feeder.ProducibleData;
+import uk.ac.ebi.ena.readtools.loader.common.feeder.ProducibleDataChecker;
 
 public class 
 DataSpot implements Serializable

```

Comment:
Please remove any unused data members after removing unused methods (see below).

Reply:
All the methods you pointed are important and are being called using Java Reflection. I have added documentation notes on the two annotations above. That will help understand how all of this works together.